Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The padding value is missing a unit (e.g., px) and should be specified for proper CSS syntax.